App.tsx

```diff
@@ -309,11 +314,66 @@ const App = () => {
               color="#841584"
             />
             <Text style={styles.debug}>{JSON.stringify(travelTime)}</Text>
-            <Button
-              onPress={() => onShare(destinationTextSaved)}
-              title="Share"
-              color="#841584"
-            />
+            {travelTime.walking !== undefined ? (
+              <Button
+                onPress={() =>
+                  onShare(
+                    `I need ${formatTravelTime(
+                      JSON.stringify(travelTime.walking)
+                    )} to walk to ${destinationTextSaved}.`
+                  )
+                }
+                title={`🚶 ${formatTravelTime(
+                  JSON.stringify(travelTime.walking)
+                )}`}
+                color="#841584"
+              />
+            ) : null}
+            {travelTime.bicycling !== undefined ? (
+              <Button
+                onPress={() =>
+                  onShare(
```

Comment:
The name of the share method irritates me. Why the `on`? We should get rid of that. 😄